schedulers/terraform/argo-workflow/karpenter-provisioners/spark-compute-optimized-provisioner.yaml

```diff
@@ -1,109 +1,111 @@
-apiVersion: karpenter.sh/v1alpha5
```

Comment:
This is new way of deploying Karpenter provisioners which uses a local Helm Chart from EKS Data Add-ons
https://github.com/awslabs/data-on-eks/blob/1e4b51971bed042adf17cb7995cceafb7790a150/analytics/terraform/spark-k8s-operator/addons.tf#L180C1-L184C1

If you have bandwidth, then you can leverage the above approach or raise another PR for the upgrade. I am fine with either approach.

Reply:
I have seen this as well. I can use this approach.